File: 1-Beginner/1013/bee1013.java
// 1013 - O Maior
import java.io.IOException;
import java.util.Scanner;

class bee1013 {

    public static void main(String[] args) throws IOException {
        Scanner scan = new Scanner(System.in);

        int a, b, c, maior;
        a = scan.nextInt();
        b = scan.nextInt();
        c = scan.nextInt();

        maior = (a+b + Math.abs(a-b))/2;
        maior = (maior+c + Math.abs(maior-c))/2;

        System.out.println(maior + " eh o maior");

        scan.close();
    }
}

File: 1-Beginner/1013/bee1013.cpp
// 1013 - O Maior
#include <iostream>
#include <cmath>

int main(void) {

    int a, b, c, maior;
    std::cin>>a>>b>>c;

    maior = (a+b + abs(a-b))/2;
    maior = (maior+c + abs(maior-c))/2;

    std::cout<<maior<<" eh o maior"<<std::endl;
}